package-lock.json

```diff
@@ -26843,6 +27671,125 @@
         "node": ">=20"
       }
     },
+    "packages/malloy-db-publisher": {
+      "name": "@malloydata/db-publisher",
+      "version": "0.0.261",
```

Comment:
This will need to be manually bumped to match the version in lerna.json before landing or shortly thereafter.